helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml

```diff
@@ -71,6 +71,12 @@ generic-service:
     FEATURE_EVENT_DELETEALL: false
     FEATURE_EVENT_ADJUDICATION_REPORT_CREATED: false
     FEATURE_EVENT_ADJUDICATION_DAMAGES_UPDATED: false
```

Comment:
just made this all consistent across ADJUDICATIONS - although everything is double typed an over the top with both defaults and entries for each environment. I suspect we could just drop the prod defaults but at least this is clear what to do when we go live